lib/output.js

```diff
@@ -1555,7 +1555,7 @@ function OutputStream(options) {
     DEFPRINT(AST_Try, function(self, output) {
         output.print("try");
         output.space();
-        print_braced(self, output);
+        print_braced(self.body, output);
```

Comment:
Ditto.

Reply:
I had to make AST_Block printable but that will be useful for later.